src/compiler/utilities.ts

```diff
@@ -3848,6 +3874,7 @@ namespace ts {
             || kind === SyntaxKind.TrueKeyword
             || kind === SyntaxKind.SuperKeyword
             || kind === SyntaxKind.NonNullExpression
+            || kind === SyntaxKind.MetaProperty

```

Comment:
Do we want to just make this a `switch`? I think it's asking for it at this point.